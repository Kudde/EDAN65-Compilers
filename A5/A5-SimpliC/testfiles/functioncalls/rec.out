Fun: rawr
 > rawr

Fun: main
 > print
 > rawr

