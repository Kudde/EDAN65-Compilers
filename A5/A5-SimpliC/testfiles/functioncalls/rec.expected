Fun: rawr
Reach: { rawr }
 > rawr

Fun: main
Reach: { print rawr }
 > rawr
 > print

